### PR TITLE
Feat demo listviewbuilder navigation

### PR DESCRIPTION
**Pull request by OnderUlasKilic**

### CODE REVIEW

File: lib/demos/my_collections_demos.dart [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    ];
+  }
+}
+
+class PaddingUtility {
+  final paddingTop = const EdgeInsets.only(top: 10);
+  final paddingBottom = const EdgeInsets.only(bottom: 20);
+  final paddingGeneral = const EdgeInsets.all(20);
+  final paddingHorizontal = const EdgeInsets.symmetric(horizontal: 20);
+}
```
Comment (BBarisKilic) [inline lib/demos/my_collections_demos.dart]: Make this class (abstract final) so it wont be initialisable as constant provider helper classes should be.

```suggestion
abstract final class PaddingUtility {
  static const paddingTop = EdgeInsets.only(top: 10);
}
```

File: lib/demos/my_collections_demos.dart [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  }
+}
+
+class CollectionsModel {
+  final String imagePath;
+  final String title;
+  final double price;
+
+  CollectionsModel(
+      {required this.imagePath, required this.title, required this.price});
```
Comment (BBarisKilic) [inline lib/demos/my_collections_demos.dart]: Add comma at the end of last item.

**Review by BBarisKilic — CHANGES_REQUESTED** [review]

File: lib/demos/my_collections_demos.dart (hunk 1 above)
Comment (OnderUlasKilic) [reply lib/demos/my_collections_demos.dart]: Done.

File: lib/demos/my_collections_demos.dart (hunk 2 above)
Comment (OnderUlasKilic) [reply lib/demos/my_collections_demos.dart]: Done.

**Review by BBarisKilic — APPROVED** [review]

**Pull request merged**